migration/migration.go

```diff
@@ -275,23 +255,64 @@ func createOrUpdateCommon(typeName string, ctx context.Context, witr models.Work
 		},
 	}
 
+	return createOrUpdateType(typeName, nil, workItemTypeFields, ctx, witr, db)
+}
+
+func createOrUpdateCommon(typeName string, ctx context.Context, witr models.WorkItemTypeRepository, db *gorm.DB) error {
+	workItemTypeFields := map[string]app.FieldDefinition{}
+	extTypeName := models.SystemPlannerItem
+	return createOrUpdateType(typeName, &extTypeName, workItemTypeFields, ctx, witr, db)
+}
+
+func createOrUpdateType(typeName string, extendedTypeName *string, fields map[string]app.FieldDefinition, ctx context.Context, witr models.WorkItemTypeRepository, db *gorm.DB) error {
 	_, err := witr.Load(ctx, typeName)
 	switch err.(type) {
 	case models.NotFoundError:
-		_, err := witr.Create(ctx, nil, typeName, workItemTypeFields)
+		_, err := witr.Create(ctx, extendedTypeName, typeName, fields)
 		if err != nil {
 			return err
 		}
 	case nil:
-		fmt.Println("Work item type exists, will update/overwrite the fields only ")
-		convertedFields, err := models.TEMPConvertFieldTypesToModel(workItemTypeFields)
+		log.Printf("Work item type %v exists, will update/overwrite the fields only and parentPath", typeName)
+		path := "/"
```

Comment:
After some discussions with other people, I think we should leave this here for the moment. Desiging an API for updating work item types is a larger undertaking.